packages/graphiql-react/__mocks__/svg.jsx

```diff
@@ -1,7 +0,0 @@
-export default function MockedIcon(props) {
-  return (
-    <svg {...props}>
-      <title>mocked icon</title>
-    </svg>
-  );
-}
```

Comment:
as an improvement we no longer provide mocks for tests, but process them with same svg loader